python/tests/test_join.py

```diff
@@ -126,28 +154,28 @@ def test_column_names():
         lhs_out_columns=["data0", "key"],
         rhs_out_columns=["data1"],
     )
-    assert_frame_equal(
-        res,
-        cudf.DataFrame({"data0": [1, 2, 3], "key": [1, 2, 3], "data1": [3, 2, 1]}),
-        ignore_row_order=True,
```

Comment:
I think this test is just failing because the result has a different order and it needs to sort by the key column (fun that the gpu one passes in CI).